### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by taehoonlee**
This PR fixes some typos.

### CODE REVIEW

File: cmake/FindCUDA/FindCUDA/run_nvcc.cmake [hunk 1]
```diff
@@ -175,7 +175,7 @@ endif()
 
 # nvcc doesn't define __CUDACC__ for some reason when generating dependency files.  This
 # can cause incorrect dependencies when #including files based on this macro which is
-# defined in the generating passes of nvcc invokation.  We will go ahead and manually
+# defined in the generating passes of nvcc invocation.  We will go ahead and manually
```
Comment (fmassa) [inline cmake/FindCUDA/FindCUDA/run_nvcc.cmake]: How should we handle PRs to external libs like FindCUDA? Modifying it here will lead to divergence from the master branch, except if we send the PR there.
This has already happened in https://github.com/pytorch/pytorch/pull/2254 for example

Comment (fmassa) [inline cmake/FindCUDA/FindCUDA/run_nvcc.cmake]: If you revert this change, I think this PR is good to go

Comment (taehoonlee) [inline cmake/FindCUDA/FindCUDA/run_nvcc.cmake]: I agree with the feedback, thank you @fmassa! Updated and Rebased.

File: torch/lib/nccl/src/common_coll.h [hunk 2]
```diff
@@ -17,7 +17,7 @@ static ncclResult_t PointerCheck(const void* pointer, struct ncclComm* comm, con
     return ncclInvalidDevicePointer;
   }
   if (attr.memoryType == cudaMemoryTypeDevice && attr.device != comm->cudaDev) {
-    WARN("%s : %s allocated on device %d mismatchs with NCCL device %d \n", opname, ptrname, attr.device, comm->cudaDev);
+    WARN("%s : %s allocated on device %d mismatches with NCCL device %d \n", opname, ptrname, attr.device, comm->cudaDev);
```
Comment (fmassa) [inline torch/lib/nccl/src/common_coll.h]: I missed this one as well. Could you remove this fix in `nccl` to simplify the merge process?
Actually, @soumith is that right? Is it easy to keep both torch and pytorch in sync with PRs in pytorch touching several repos in torch?

**Comment by fmassa** [comment]
@taehoonlee To be sure this PR can be merged without any difficulties, I'd remove as well the changes in `TH` and `THCUNN`, so that all the changes are inside this repo, and then it's definitely easy to merge this without having merge difficulties afterwards

**Comment by taehoonlee** [comment]
Thank you for the comments @fmassa! Updated.

**Review by fmassa — APPROVED** [review]
Looks good to me now, thanks!

**Comment by soumith** [comment]
thanks a lot Taehoon!
